internal/p2p/uploads.go

```diff
@@ -40,24 +40,24 @@ func ProcessUploads(folderPath string) {
 						fmt.Println("[INFO] File Security checking file: " + f + " with SHA256: " + fsha256)
 						var result bool
 						var err error
-						// TODO: 				if appconfInstance.SecurityCapability [1 2 3 4]
-						if appconfInstance.SecurityCapability <= 3 {
+
+						// Skip all security scanning if FFSkipVT is enabled
+						if appconfInstance.FFSkipVT {
```

Comment:
It seems like we've gotten deeply nested here. Let's consider refactoring this to be easier to read.